sample_kbs/src/main.rs

```diff
@@ -18,11 +18,11 @@ async fn main() -> Result<()> {
         .version("1.0.0")
         .arg(
             Arg::with_name("socket addr")
-                .long("grpc_sock")
+                .long("keyprovider_sock")
```

Comment:
Since the names are changing, is it too late to consider replacing the underscore with a dash? It looks better imho and is the expected way to specify long options (`--keyprovider-sock`).

Same comment for `--getresource_sock` (rename to `--getresource-sock`).

Reply:
I think the goal of this PR is to have the the names used by the `sample_kbs` be the same as the names used by the Attestation Agent itself. Maybe we can have a separate issue for whether we want to change the argument name across the entire project?

Reply:
> I think the goal of this PR is to have the the names used by the `sample_kbs` be the same as the names used by the Attestation Agent itself. Maybe we can have a separate issue for whether we want to change the argument name across the entire project?

@jodh-intel @fitzthum Yes, If we decide to change the parameter name, Kata-agent also needs to make changes synchronously. Maybe it's better to open a new PR to solve this problem, if you want to do so.